TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/Models/PushFactor.swift

```diff
@@ -35,6 +35,22 @@ struct PushFactor: Factor, Codable {
 struct Config: Codable {
   let credentialSid: String
   var notificationPlatform: NotificationPlatform = .apn
```

Comment:
since you are setting a default value through the constructor, it is necessary to have a default value on the var declaration? I think this can be `let` and the `= .apn` be removed

Reply:
Done, thanks